### PR TITLE
DOCSP-34852 Support $out

### PR DESCRIPTION
**Pull request by jordan-smith721**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-34852
Staging - https://preview-mongodbjordansmith721.gatsbyjs.io/csharp/DOCSP-34852-support-out/fundamentals/builders/#write-pipeline-results-to-a-collect

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/builders.txt [hunk 1]
```diff
@@ -335,6 +340,57 @@ as a ``BsonDocument`` to the `AppendStage() method
...
+   var pipeline = new EmptyPipelineDefinition<Flower>()
+       .Match(matchFilter)
+       .Out(outputCollection);
+
+You can write the results of an aggregation pipeline to a time series collection
+by specifying a ``TimeSeriesOption`` object and passing it as the second
+parameter to the ``Out()`` method.
+
+Consider the documents in the collection contain a ``datePlanted`` field that
+holds BSON date values. You can store the documents in this collection in a time
```
Comment (mongoKart) [inline source/fundamentals/builders.txt:373]: S: this reads a little funny

```suggestion
Imagine that the documents in the [which collection?] collection contain a ``datePlanted`` field that
holds BSON date values. You can store the documents in this collection in a time
```

File: source/fundamentals/builders.txt [hunk 2]
```diff
@@ -335,6 +340,57 @@ as a ``BsonDocument`` to the `AppendStage() method
...
+series collection by using the ``datePlanted`` field as the time field.
+
+The following example creates a ``TimeSeriesOptions`` object and specifies
+``datePlanted`` as the ``timeField``. It then builds an aggregation pipeline that matches all documents
+with a ``season`` field value of ``"Spring"`` and outputs them to a
+time series collection called ``"springFlowerTimes"``.
```
Comment (mongoKart) [inline source/fundamentals/builders.txt:379]: nit: i don't think we usually refer to collection names using quotes, the way we do with field values. i'd remove them here, even though the code sample uses quotes.

```suggestion
time series collection called ``springFlowerTimes``.
```

**Review by mongoKart — APPROVED** [review]
LGTM plus a couple small things

File: source/fundamentals/builders.txt (hunk 1 above)
Comment (jordan-smith721) [reply source/fundamentals/builders.txt]: Correct. I wasn't a huge fan of this wording either, I think yours gets the message across better!

**Review by adelinowona — APPROVED** [review]
LGTM!

**Pull request merged**